bolt/org_config.go

```diff
@@ -0,0 +1,169 @@
+package bolt
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/influxdata/chronograf"
+	"github.com/influxdata/chronograf/bolt/internal"
+)
+
+// Ensure OrganizationConfigStore implements chronograf.OrganizationConfigStore.
+var _ chronograf.OrganizationConfigStore = &OrganizationConfigStore{}
+
+// OrganizationConfigBucket is used to store chronograf organization configurations
+var OrganizationConfigBucket = []byte("OrganizationConfigV1")
+
+// OrganizationConfigStore uses bolt to store and retrieve organization configurations
+type OrganizationConfigStore struct {
+	client *Client
+}
+
+func (s *OrganizationConfigStore) Migrate(ctx context.Context) error {
+	return nil
+}
+
+// FindOrCreate gets an OrganizationConfig from the store or creates one if none exists for this organization
+func (s *OrganizationConfigStore) FindOrCreate(ctx context.Context, orgID string) (*chronograf.OrganizationConfig, error) {
```

Comment:
Where is the `Create` part of this method?

Reply:
Thank you, I missed that part

Reply:
Fixed.